src/index/FTSAlgorithms.cpp

```diff
@@ -373,6 +378,73 @@ void FTSAlgorithms::aggScoresAndTakeTopKContexts(
              << " entity-score-context tuples now.\n";
 }
 
+// _____________________________________________________________________________
```

Comment:
Do we still need the "old" functions that don't take `WordEntityPostings` ? If yes, for what?

Reply:
I don't think so.